src/docs/Foundations/Typography/BodyStyles/design.mdx

```diff
@@ -3,10 +3,4 @@ title: 'Design'
 order: 3
 ---
 
-##What's included ?
-
-| **[BA] Bodies.sketch** |                 |
-| ---------------------- | --------------- |
-| All viewports :        | 01-Small--14    |
-|                        | 02-Medium--16px |
-|                        | 03-Large--18px  |
+##What's included ?
```

Comment:
Shouldn't we delete the "design" tab given there's no content on it?

Reply:
@tiagoux this ticket is still in "Waiting for pooling" because I have to see this point (and some others) with the designers